compose/sample-apps.yml

```diff
@@ -95,7 +95,7 @@ services:
       start_period: 30s
       timeout: 5s
   quarkus-test-agent:
-    image: ${QUARKUS_TEST_IMAGE:-quay.io/andrewazores/quarkus-test:latest}
+    image: ${QUARKUS_TEST_IMAGE:-quay.io/miwan/quarkus-test:latest}
```

Comment:
This will need to be put back, or else we need to get around to finally publishing this testing application image to some repository belonging to our org.

Reply:
it would be nice to set up the new sample app repo so we can merge my other -agent [PR](https://github.com/cryostatio/cryostat-agent/pull/397) too